src/main/java/net/pms/dlna/DLNAImageProfile.java

```diff
@@ -0,0 +1,341 @@
+/*
+ * Universal Media Server, for streaming any medias to DLNA
```

Comment:
Nop - but I don't actually READ the header every time, I just copy it ;)

Will fix them all.